zha/application/platforms/__init__.py

```diff
@@ -205,6 +207,13 @@ def translation_key(self) -> str | None:
             return self._attr_translation_key
         return None
 
+    @property
+    def translation_placeholders(self) -> dict[str, str] | None:
+        """Return the translation placeholders."""
+        if hasattr(self, "_attr_translation_placeholders"):
+            return self._attr_translation_placeholders
+        return None
```

Comment:
Removing the `hasattr` check per my comment here: https://github.com/zigpy/zha/pull/533#discussion_r2361297687
```suggestion
        return self._attr_translation_placeholders
```